snuba/datasets/configuration/events_analytics_platform/storages/eap_items_log.yaml

```diff
@@ -225,13 +225,6 @@ query_processors:
       prewhere_candidates:
         [trace_id]
   - processor: TupleUnaliaser
-  - processor: ClickhouseSettingsOverride
-    args:
-      settings:
-        max_memory_usage: 5000000000
-        max_rows_to_group_by: 1000000
-        group_by_overflow_mode: any
```

Comment:
the default for these settings is unlimited:

https://clickhouse.com/docs/operations/settings/settings#max_memory_usage
https://clickhouse.com/docs/operations/settings/settings#max_rows_to_group_by

you'd be making this less restricted, not more

Reply:
I'm not trying to restrict, I'm trying to remove EAP specific settings.